packages/api/src/external/aws/appConfig.ts

```diff
@@ -5,14 +5,17 @@ import { capture } from "../../shared/notifications";
 
 export async function getCxsWithEnhancedCoverageFeatureFlagValue(): Promise<string[]> {
   try {
-    const cxIds = await getFeatureFlagValue<string[]>(
+    const featureFlag = await getFeatureFlagValue<{
+      enabled: boolean | undefined;
+      cxIds: string[] | undefined;
+    }>(
       Config.getAWSRegion(),
       Config.getAppConfigAppId(),
       Config.getAppConfigConfigId(),
       Config.getEnvType(),
       Config.getCxsWithEnhancedCoverageFeatureFlagName()
     );
-    return cxIds ? cxIds : [];
+    if (featureFlag?.enabled) return featureFlag?.cxIds ? featureFlag?.cxIds : [];
```

Comment:
NABD, but could simplify the combination of possible outcomes by

```ts
if (featureFlag?.enabled && featureFlag?.cxIds) return featureFlag.cxIds;
```

Reply:
yay for simpler logic - done 👍